oncotext/datasets/pathology_dataset_classifier.py

```diff
@@ -15,14 +15,12 @@
 random.seed(0)
 
 
-class PathologyDataset(data.Dataset):
+class PathologyDatasetClassifier(data.Dataset):
```

Comment:
Would rename to "PathologyClassificationDataset" 
It's primarily a dataset, not a classifer

Reply:
resolved